openssl/src/asn1.rs

```diff
@@ -512,6 +511,23 @@ impl Asn1Integer {
     }
 }
 
+impl Ord for Asn1Integer {
+    fn cmp(&self, other: &Self) -> Ordering {
+        Asn1IntegerRef::cmp(self, other)
+    }
+}
+impl PartialOrd for Asn1Integer {
+    fn partial_cmp(&self, other: &Asn1Integer) -> Option<Ordering> {
+        Some(self.cmp(other))
```

Comment:
Couldn't you just use `Asn1IntegerRef::partial_cmp` here as well?

Reply:
I could, but I couldn't see it making any difference and I'd already written this code.

Reply:
So the stuff looks more similar.
yeah, its more a nit.